### ISSUE TITLE
install error on 1.12

### ISSUE THREAD

**Issue by Tecko71** [foreverguest/repository.foreverguest#15]
Have an install error on version 1.12, on kodi android tv. version 1.9.3 works perfect. Can not find the logs.

**Comment by Tecko71** [comment]
version 1.11 login error. Version 1.10.6 works also.

**Comment by foreverguest** [comment]
Which Kodi version are you using?

**Comment by Tecko71** [comment]
Ik was using old based kodi apps. Now I have 1.12 running now on kodi 18.4 (nvidea shield).
Youre 2.0beta2 is not working for me.
Everything is woring with 1.12 accept browsing albums in artist folder (only browsing saved albums)

I finally found a log file hope you can use it

[kodi.log](https://github.com/foreverguest/repository.foreverguest/files/3656981/kodi.log)

second log install errors version 2.0beta2:

[kodi (1).log](https://github.com/foreverguest/repository.foreverguest/files/3657037/kodi.1.log)

after installing the betta I can not go back to 1.12, gpm does not store the inlog anymore. (clearing kodi cahce solved the problem, back to 1.12 again)

**Comment by foreverguest** [comment]
I'm not supporting version 1.12 anymore, so let's try to figure the beta problem.

In the logs I can see 1.12 was able to identify you as a Play Music subscriber, but not the beta. Are you sure you were logged with the same user in Google when you authorized the beta addon?

**Comment by Tecko71** [comment]
I did activate wrongly. But now gpm wont play streams. Artist and albums are in my libary (but only saved albums not the catholog of artists). Kodi 19,  gpm 2.0beta2.

I created a log for the stream error:

[kodi (4).log](https://github.com/foreverguest/repository.foreverguest/files/3665233/kodi.4.log)

**Comment by foreverguest** [comment]
I have found some problems, thanks for reporting.

Please try new version: https://app.box.com/s/5qd531m7akjgmvy2eauk7uyqkvzyy3zi

**Comment by Tecko71** [comment]
Yes it works now! thx:)     ...only I have one request, can you at in artists libary; catalog browsing artist albums and not only the saved ones?

**Comment by Tecko71** [comment]
I find a issue with updatinge libary I got an error, that is the only problem I have now. Log file:
[kodi3009.log](https://github.com/foreverguest/repository.foreverguest/files/3673536/kodi3009.log)

**Comment by foreverguest** [comment]
> 
> 
> Yes it works now! thx:) ...only I have one request, can you at in artists libary; catalog browsing artist albums and not only the saved ones?

Not possible, there is no api call to list all artists in catalog, you'll have to use the 'Search'.

**Comment by foreverguest** [comment]
> 
> 
> I find a issue with updatinge libary I got an error, that is the only problem I have now. Log file:
> [kodi3009.log](https://github.com/foreverguest/repository.foreverguest/files/3673536/kodi3009.log)

OK, fixed this one too: https://app.box.com/s/7whrlps3a8qy75kg08eb07a55svaf97u

**Comment by Tecko71** [comment]
Oke thx good work !  I have only error when browsing stations and the new stations section. But that is not a problem for me. And a error when I push the home button. If it has to be working I can add a log again.

**Comment by foreverguest** [comment]
Please, send your log, let's try to fix all problems.

**Comment by Tecko71** [comment]
Here the log with browsing stations and home button: 

[kodi0110.log](https://github.com/foreverguest/repository.foreverguest/files/3678343/kodi0110.log)

**Comment by foreverguest** [comment]
New version https://app.box.com/s/fnndlx8h23me7zkjriq9ykzezmpj5e3h

I have added a debug log to the stations browsing, it should not fail, please post a new log when possible.

**Comment by Tecko71** [comment]
Today's hits and new music error.
Home button works 

[kodi 0310.log](https://github.com/foreverguest/repository.foreverguest/files/3684806/kodi.0310.log)

**Comment by foreverguest** [comment]
Hope everything is working now:
https://app.box.com/s/gw279lavlz9r0eurq4970xa09ov4fjel

**Comment by Tecko71** [comment]
yes it is working! no errors anymore. Now good for release 2.0 in repo?
